.github/workflows/update-pinned-reqs.yml

```diff
@@ -2,7 +2,7 @@ name: Update pinned requirements
 
 on:
   schedule:
-  - cron: 37 11 * * 5
+  - cron: 37 11 * * 4
```

Comment:
Fixing an off-by-one error from #3041.  It must be Thursday. I never could get the hang of Thursdays.